### PR TITLE
clarify law testing's dependency on cats-testkit and scalatest

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

File: docs/src/main/tut/typeclasses/lawtesting.md [hunk 1]
```diff
@@ -7,23 +7,25 @@ section: "typeclasses"
 # Law testing
 
 [Laws](https://typelevel.org/cats/typeclasses.html#laws) are an important part of cats.
-Cats uses `catalysts` and `discipline` to help test instances with laws.
-To make things easier, cats ships with `cats-testkit`, which makes use of `catalysts` and `discipline` and exposes `CatsSuite` based on ScalaTest.
+Cats uses `catalysts` and [discipline](https://github.com/typelevel/discipline) to help test instances with laws. To test type class
```
Comment (tpolecat) [inline docs/src/main/tut/typeclasses/lawtesting.md]: Why is `catalysts` relevant here?

File: docs/src/main/tut/typeclasses/lawtesting.md [hunk 2]
```diff
@@ -50,8 +52,8 @@ For simplicity we'll just use `Eq.fromUniversalEquals`:
 implicit def eqTree[A: Eq]: Eq[Tree[A]] = Eq.fromUniversalEquals
 ```
 
-Then we can begin to write our law tests. Start by creating a new class in your `test` folder and inheriting from `cats.tests.CatsSuite`.
-`CatsSuite` extends the standard ScalaTest `FunSuite` as well as `Matchers`.
+Then we can begin to write our law tests. If you are using `ScalaTest` you can inherit from `cats.tests.CatsSuite` 
+from `cats-testkit`. `CatsSuite` extends the standard ScalaTest `FunSuite` as well as `Matchers`.
```
Comment (tpolecat) [inline docs/src/main/tut/typeclasses/lawtesting.md]: What do you do if you're not using ScalaTest? Not sure this helps.

**Comment by kailuowang** [comment]
@tpolecat how about now?

File: docs/src/main/tut/typeclasses/lawtesting.md [hunk 3]
```diff
@@ -7,23 +7,28 @@ section: "typeclasses"
 # Law testing
 
 [Laws](https://typelevel.org/cats/typeclasses.html#laws) are an important part of cats.
-Cats uses `catalysts` and `discipline` to help test instances with laws.
-To make things easier, cats ships with `cats-testkit`, which makes use of `catalysts` and `discipline` and exposes `CatsSuite` based on ScalaTest.
+Cats uses [discipline](https://github.com/typelevel/discipline) to define type class laws and 
+the [ScalaCheck](https://github.com/rickynils/scalacheck) tests based on them.
+
+To test type class laws from Cats against your instances, you need to add a `cats-laws` dependency. 
+If you are using `ScalaTests`, Cats also ships with optional `cats-testkit`, which provites a convenient
```
Comment (LukaJCB) [inline docs/src/main/tut/typeclasses/lawtesting.md]: typo `ScalaTests`

**Comment by kailuowang** [comment]
made some tut compilation corrections. need a quick re-review.

File: docs/src/main/tut/typeclasses/lawtesting.md [hunk 4]
```diff
@@ -43,34 +48,27 @@ object Tree {
   }
 }
 ```
-We will also need to create an `Eq` instance, as most laws will need to compare values of a type to properly test for correctness.
-For simplicity we'll just use `Eq.fromUniversalEquals`:
-
-```tut:book
-implicit def eqTree[A: Eq]: Eq[Tree[A]] = Eq.fromUniversalEquals
+```tut:invisible
+import Tree._ //there is no real companion object is in REPL
```
Comment (rossabaker) [inline docs/src/main/tut/typeclasses/lawtesting.md]: s/is in/in the/

**Pull request merged**